.github/workflows/build-test.yml

```diff
@@ -28,4 +28,4 @@ jobs:
       - name: test
         run: make test
       - name: lint
-        run: sudo make lint
+        run: make lint
```

Comment:
@karencfv I didn't see why the `sudo` was added in the blamed commit (https://github.com/oxidecomputer/terraform-provider-oxide/pull/95). I removed it because it was causing the runner to fall back to the version of Go included in the runner rather than the version of Go installed by `setup-go`.